lexnlp/extract/pt/regulations.py

```
@@ -0,0 +1,342 @@
+                  
+                  Parameters:
+                      name (str): Canonical label for the regulation (e.g., act name or trigger phrase).
+                      coords (tuple[int, int]): Start and end character offsets of the match in the source text.
+                      surface (str): Exact matched text to store in the annotation's `text` field.
+                      locale (str): Locale or language code associated with the source text (e.g., "pt").
+                      country (str): Country associated with the regulation; defaults to "Brazil".
+                  
+                  Returns:
+                      RegulationAnnotation: Annotation populated with the provided name, coords, text, locale, and country.
+                  """
+                  return RegulationAnnotation(
```

Pylint (4.0.5)

[error] Parsing failed: 'unexpected indent (lexnlp.extract.pt.regulations, line 173)'

(E0001)